cpp/cmake_modules/Findutf8proc.cmake

```
@@ -15,6 +15,31 @@
 # specific language governing permissions and limitations
 # under the License.
 
+function(EXTRACT_UTF8PROC_VERSION)
```

Could you use lower case because we use lower case for function call?

```suggestion
function(extract_utf8proc_version)
```

---

The suggestion was applied!